docs/content/contribution-guidelines/operations/document-management.md

```diff
@@ -148,10 +152,11 @@ You can view and modify our documentation in the branch you have checked out by
 pip install -r requirements.txt
 ```
 
-Alternatively, use the following commands to use the Makefile target to construct an adequate virtual environment and enter it.
+Alternatively, use the following commands to use the Makefile target to construct an adequate virtual environment and enter it. From the root of the repository(where the Makefile is), run:
 
 ```shell
-( cd ..; make venv )
```

Comment:
Note that we tend to follow a certain design style for documents like this. We know that readers tend to skip over the text --- not necessarily on the first run through, but when they do it over and over again --- and just copy-and-paste the exhibited commands into a shell. We try to make that as easy and correct as possible.

This change introduces a sharp edge: it requires a user who is reading along and issuing the displayed commands to inject a command that is not displayed --- to `cd` to the repo root directory.

Reply:
Let's leave the sub-shell usage in place.